lmp/tokenizer/_base_dict_tokenizer.py

```diff
@@ -370,7 +430,27 @@ def build_vocab(
             min_count:
                 Minimum of token's frequence. If token's frequence is smaller
                 than `min_count`, then discard that token.
+         Raises:
+            TypeError:
+                When `batch_sequences` is not instance of `Iterable[str]` or
+                `min_count` is not instance of `int`.
         """
+        # Type check.
```

Comment:
Perform type check on `batch_sequence_to_tokens` already.
Use `try ... except` block to catch error instead